src/main/java/com/ss/training/lms/controller/LibrarianController.java

```diff
@@ -117,47 +121,67 @@
 
     }
 
-    @PutMapping(value = "/lms/librarian/branches/{branchId}/books/{bookId}")
-    public ResponseEntity<String> updateCopies(@PathVariable int branchId,
+    @PutMapping(value = "/lms/librarian/branches/{branchId}/books/{bookId}",
+            produces = {
+                    MediaType.APPLICATION_JSON_VALUE,
+                    MediaType.APPLICATION_XML_VALUE
```

Comment:
The media types aren't needed with a pom dependency. I think Justin found it. The media types don't hurt, but removing them could clean up the code.

Reply:
oh awesome. ok. I'll remove them then.